src/core/overlay/overlay-container.ts

```diff
@@ -1,12 +1,21 @@
 
 
-/**
- * Create the overlay container element, which is simply a div
- * with the 'md-overlay-container' class on the document body.
- */
-export function createOverlayContainer(): Element {
-  let container = document.createElement('div');
-  container.classList.add('md-overlay-container');
-  document.body.appendChild(container);
-  return container;
+export class OverlayContainer {
+  private _containerElement: Element;
+
+  getContainerElement(): Element {
```

Comment:
Add function description that includes the fact that this method will lazily create the container.